MetamodelLD.md

```diff
@@ -266,7 +266,7 @@ Deze paragraaf is een aanvulling op de paragraaf [[[#specificatie-metagegevens]]
 | toelichting | [`mim:toelichting`](http://bp4mc2.org/def/mim#toelichting) | owl:DatatypeProperty | [grondslag](#metagegeven-toelichting)
 | type | [`mim:type`](http://bp4mc2.org/def/mim#type) | owl:ObjectProperty | [grondslag](#metagegeven-type) |
 | unidirectioneel | [`mim:unidirectioneel`](http://bp4mc2.org/def/mim#unidirectioneel) | owl:DatatypeProperty | [grondslag](#metagegeven-unidirectioneel) |
-| Modelelementidentificatie | [`mim:modelelementIdentificatie`](http://bp4mc2.org/def/mim#modelelementIdentificatie) | owl:DatatypeProperty | [grondslag](#metagegeven-modelelement-identificatie) |
+| modelelementidentificatie | [`mim:modelelementidentificatie`](http://bp4mc2.org/def/mim#modelelementidentificatie) | owl:ObjectProperty | [grondslag](#metagegeven-modelelement-identificatie) |
```

Comment:
In hoofdstuk 2 heet dit "URI". Ik meende dat het de bedoeling was om dit te veranderen in "identificatie". Daarmee zou sprake zijn van een eigenschap "identificatie". De term "modelelementidentificatie" verwijst m.i. niet naar een eigenschap, maar naar een propertyshape ("modelelement.identificatie"). We hebben bv ook niet modelelementnaam ofzo.

Overigens is de vraag of er een verschil is tussen de URI van het modelelement in MIM en de URI van een modelelement in LD. Wat bedoelen we met de identificatie van een modelelement? Wat mij betreft mag deze laatste vraag een onderwerp zijn voor MIM 2.0...

Reply:
eens dat het onduidelijk was en is. Er is nu gekozen voor modelelementidentificatie ipv URI.

Reply:
Waarom niet gewoon "identificatie"? Wat is de rationale erachter om (anders dan bij naam, alias, definitie, etc, etc, etc) nu expliciet te maken dat het (alleen) van toepassing is op een modelelement?